server/channels/api4/shared_channel_test.go

```diff
@@ -165,7 +165,12 @@ func TestCreateDirectChannelWithRemoteUser(t *testing.T) {
 		th := setupForSharedChannels(t).InitBasic()
 		defer th.TearDown()
 		client := th.Client
-		defer client.Logout(context.Background())
+		defer func() {
+			_, err := client.Logout(context.Background())
+			if err != nil {
+				require.NoError(t, err)
+			}
```

Comment:
While the code is not wrong, there is no need to check if `err != nil`. `require.NoError` does that for you.

Reply:
Done!